plugin-directorymenu/directorymenuconfiguration.ui

```diff
@@ -40,6 +33,13 @@
         </property>
        </widget>
       </item>
+      <item row="0" column="0">
```

Comment:
Why did you move this section?

Reply:
I believe that this inadvertently happened when I was editing the rest of the file in Qt Creator. I moved that item back to its original spot.